js/classes/entities/midground/chad.js

```diff
@@ -318,7 +354,7 @@ class Chad {
             }
 
             // release wind particles every 0.05 seconds
```

Comment:
This comment is now outdated and if we update the magic numbers to clear variables it is unnecessary.